pkg/cmd/triggertemplate/describe.go

```diff
@@ -0,0 +1,185 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package triggertemplate
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/cli/pkg/printer"
+	"github.com/tektoncd/cli/pkg/validate"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .TriggerTemplate.Name }}
+{{decorate "bold" "Namespace"}}:	{{ .TriggerTemplate.Namespace }}
+
+{{decorate "params" ""}}{{decorate "underline bold" "Params\n"}}
+
+{{- if eq (len .TriggerTemplate.Spec.Params) 0 }}
+ No params
+{{- else }}
+ NAME	TYPE	DESCRIPTION	DEFAULT VALUE
+{{- range $p := .TriggerTemplate.Spec.Params }}
+{{- if not $p.Default }}
+{{- if eq $p.Type "" }}
```

Comment:
Added extra check for params `Type` because like `Pipeline` `Triggers` doesn't have defaults set for Params 
raised an [issue](https://github.com/tektoncd/triggers/issues/549)  in order to track once trigger allow defaults for `Type` then will modify here